src/glossary.md

```diff
@@ -130,6 +140,14 @@ This is not affected by applied type arguments. `struct Foo` is considered local
 `Vec<Foo>` is not. `LocalType<ForeignType>` is local. Type aliases do not
 affect locality.
 
+### Module
+
+A module is a container for zero or more [items]. Modules are organized in a
+tree, starting from an unnamed module at the root called the crate root or the
+root module. [Paths] may be used to refer to items outside of a module, which
```

Comment:
"outside of the module that defines them"? "from other modules"?

Reply:
Yea, the sentence was quite awkward. I went with your second suggestion, thanks!